code/game/turfs/turf.dm

```diff
@@ -45,6 +45,10 @@
 	/// How pathing algorithm will check if this turf is passable by itself (not including content checks). By default it's just density check.
 	/// WARNING: Currently to use a density shortcircuiting this does not support dense turfs with special allow through function
 	var/pathing_pass_method = TURF_PATHING_PASS_DENSITY
+	/// Are we a explodable turf?
+	var/explodable = FALSE
+	/// Do we have a explodable overlay?
+	var/explodable_overlay
```

Comment:
I've done a little bit of looking, it seems like you can move it up to walls, it would just need an istype check on the helper and component. I really think we should do this so that we don't add 2 vars to EVERY turf in game

Reply:
I'll look into it, I recall having icon shenanigins when it was on walls

Reply:
done